external_review/export_DO.sh

```diff
@@ -5,9 +5,9 @@ function export_DO {
     echo $2
     geography_level=$1
     category=$2
-    filename="${category}_${geography_level}.csv"
+    filename="${category}_by_${geography_level}.csv"
```

Comment:
Remove "_by_"

Reply:
yep, good catch that change will be made. note that the `mc cp ...` changes the name of the file and what's in DO doesn't have the _by.